docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -504,10 +517,10 @@ spec:
         readOnly: true
 "
 ----
-+
-* Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
```

Comment:
Should we keep the note about bridge mappings?

Reply:
indeed, looks like it got lost during copy-pasta?

Reply:
or no, it was moved https://github.com/openstack-k8s-operators/data-plane-adoption/pull/475/files#diff-603a706779fa17d06015eb6775886bdaa06a6531c6dc1a2a4ee2edbb4f5e6c66R236